curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f46e7a4750dd05b5a673920.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-87
 
 # --description--
 
-Now apply the `address` class to the `p` element containing the address.
+Now apply the address class to the p element containing the street address 123 Free Code Camp Drive
```

Comment:
You shouuldn't remove the backticks for the address class and p element. 
Those still need to be there.
Also, you should add the period at the end. Since it is a sentence it should have a period.

Reply:
Thanks for the feedback. I've made these changes.